Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -1602,6 +1611,10 @@ void RenderLayer::updateDescendantDependentFlags()
 
 bool RenderLayer::computeHasVisibleContent() const
 {
+#if ENABLE(LAYER_BASED_SVG_ENGINE)
+    if (renderer().document().settings().layerBasedSVGEngineEnabled() && lineageOfType<RenderSVGHiddenContainer>(renderer()).first())
```

Comment:
Should this have a FIXME for making it more efficient if needed? For example, maybe we can have a flag somewhere whether the svg subtree had a hidden container at all.

Reply:
Done, here and in RenderLayerBacking.

Reply:
Thanks. I do worry that this slows the non SVG codepath, can you check with Simon? Or before LBSE the FIXME should be implemented...

Reply:
Maybe I miscommunicated this, but I plan to address this soon, just not in this PR. Adding the flag and correctly updating it requires some boiler plate stuff that I want to avoid including here.

Reply:
Ok. BTW, pretty sure it is up to the reviewer to mark the conversation as resolved :P

Reply:
Heh. I thought these were information for me (that I can mark them as answered for myself). Now I realized the true purpose of that button :-) :-) Makes sense, will leave it in future.